### DISCUSSION TITLE
How to start a new Backup?

### DISCUSSION THREAD

**Discussion by Schema-F** [nextcloud/all-in-one#1427] [Questions] [answered]
How can I start a complete new Backup with the built-in backup solution?

**Comment by szaimen** [comment]
I dont understand the question. Did you put in a path and clicked on create backup and now want to change the path?

**Reply by Schema-F** [reply]
Exactly, I tried automatic backups a few days on a test-hdd and now I want to change the mountpoint and start with the "productive" backup.

**Comment by szaimen** [comment, marked as answer]
You can adjust the backup location in `/var/lib/docker/volumes/nextcloud_aio_mastercontainer/_data/data/configuration.json`. Additionally you need to delete the file `/var/lib/docker/volumes/nextcloud_aio_mastercontainer/_data/data/borg.config` which should allow you to start over.

**Reply by Schema-F** [reply]
Thanks for your help. 
I moved the borg.conf to a temporary folder and changed the backup directory. Afterwards backup to the new directory worked perfect, but there weren't any options to configure daily backups in the aio-interface anymore.
So i tried to move back the borg.conf to its place in the docker volume. Now all config options are back and everything works fine.
Nextcloud-aio-team does a really great job!

**Reply by szaimen** [reply]
> Nextcloud-aio-team does a really great job!

Thats always great to hear, thanks! :)
